Epam.TAF.Web/PageObgects/Pages/JoinOurTeamPage.cs

```diff
@@ -0,0 +1,174 @@
+
+        private const string _skillsFieldXPath = "//*[contains(@class, 'selected-params')]";
+        private const string _skillsDropDownXPath = "//*[@class= 'multi-select-dropdown']";
+        private const string _checkBoxesSkillsInDropDownXPath = "//*[@class= 'checkbox-custom-label']";
+
+        private const string _findButtonJoinOurTeamPageXPath = "//*[contains(@class, 'recruiting-search__submit')] ";
+
+        private const string _titleWithkeywordResultsXPath = "//*[contains(@class, 'search-result__heading')]";
+        private const string _resultsOnJoinOurTeamPageXPath = "//*[@class='search-result__item']";
+
+        private const string _titleForEmptySearchResult = "//div[contains(@class, 'search-result__error-message')]";
+
+        private const string _messageForEmptyResult = "Sorry, your search returned no results. Please try another combination.";
+
+        public override string Url => PageUrls.JoinOurTeamPageUrl;
+
+        public override Title Title => new Title(By.TagName("h1"));
+
+        public Title EmptyResultTitle => new Title(By.XPath(_titleForEmptySearchResult));
+
+        public TextInput KeyWordTextInput => new TextInput(By.XPath(_keywordInputFieldXPath));
+
+        public Button FindButtonJoinOurTeamPage => new Button(By.XPath(_findButtonJoinOurTeamPageXPath));
+
+        public Title TitleWithKeywordResults => new Title(By.XPath(_titleWithkeywordResultsXPath));
+
+        public DropDown SkillsDropDownJoinOurTeam => new DropDown(By.XPath(_skillsFieldXPath));
+
+        public DropDown LocationDropDownJoinOurTeam => new DropDown(By.XPath(_locationFieldJoinOurTeamXPath));
+
+        public Panel CheckBoxesSkillsPanel => new Panel(By.XPath(_skillsDropDownXPath));
+
+        public ElementsList<CheckBox> CheckBoxSkills => new ElementsList<CheckBox>(By.XPath(_checkBoxesSkillsInDropDownXPath));
+
+        public ElementsList<Link> ResultsSkillsOnJoinOurTeam => new ElementsList<Link>(By.XPath(_resultsOnJoinOurTeamPageXPath));
+
+        public ElementsList<Link> CountriesLocationInDropDownOnJoinOurTeam => new ElementsList<Link>(By.XPath(_countriesLinkInDropDownJoinOurTeamXPath));
+
+        public bool EmptyResultTitleDisplayed()
+        {
+            return EmptyResultTitle.IsDisplayed();
+        }
+
+        public bool IsMessageForNotFoundResult()
+        {
+            var displayedMessage = EmptyResultTitle.GetText();
+            if (displayedMessage.Equals(_messageForEmptyResult)) { return true; }
+            else { return false; }
+        }
+
+        public bool IsTitleDisplayed()
+        {
+            return Title.IsDisplayed();
+        }
+
+        public bool IsTitleForPageWithResults()
+        {
+            var titleLine = GetTitleWithResult();
+
+            if (titleLine.Contains("job openings related to", StringComparison.OrdinalIgnoreCase))
```

Comment:
it is enough to return titleLine.Contains("job openings related to", StringComparison.OrdinalIgnoreCase) without if..else block.